src/cr/cube/matrix.py

```diff
@@ -528,7 +528,7 @@ def _row_elements(self):
         return self.rows_dimension.valid_elements
 
     @lazyproperty
-    def _valid_rows_idxs(self):
+    def _valid_row_idxs(self):
```

Comment:
I'm not so sure I like the new name better. Row indexes (as it is changed to) implies that we're dealing with the indexes of a row. The first line of the docstring also uses the same noun "rows". Ideally I'd put in something like "new rows' indexes", but we can't do that in python method definition. But I'll leave it to your best judgement... We also need to take into account who'll ever read this, so the new name _might_ be OK (even though I still dislike it :) ).